src/js/component.js

```diff
@@ -220,28 +220,35 @@ class Component {
    * @return {string}
    *         The localized string or if no localization exists the english string.
    */
-  localize(string) {
+  localize(string, tokens, defaultValue = string) {
```

Comment:
do we actually need "defaultValue" as an argument here? Can't we just make a copy of string in the function